### PR TITLE
Updating Blkom presence to v1.0.1

### PR DESCRIPTION
**Pull request by GhaithAlHallak8**
image
image
image
image
image

### CODE REVIEW

File: websites/A/Anime4up/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "description": {
+    "en": "Anime4up - Arabic website to watch and download Arabic subbed anime with multiple servers.",
+    "ar_SA": "انمي فور أب - موقع عربي متخصص لمشاهدة وتحميل الانمي المترجم باللغة العربية بسيرفرات متعددة."
+  },
+  "service": "Anime4up",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/qwOsx7u.png",
+  "thumbnail": "https://i.imgur.com/FfzRGF1.png",
+  "color": "#38b5eb",
+  "tags": ["embedded-links", "anime4up", "streaming"],
```
Comment (ririxichan) [inline websites/A/Anime4up/dist/metadata.json:16]: tags are not embedded-links nor anime4up

**Review by ririxichan — CHANGES_REQUESTED** [review]
upload proof of working presence (Anime4up)

**Comment by GhaithAlHallak8** [comment]
image
image
image
image
image

**Comment by GhaithAlHallak8** [comment]
image
image
image
image
image

**Review by ririxichan — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**